collectors/osd.go

```diff
@@ -698,13 +698,13 @@ func (o *OSDCollector) collectOSDPerf() error {
 		if err != nil {
 			return err
 		}
-		o.CommitLatency.WithLabelValues(osdName, lb.DeviceClass, lb.Host, lb.Rack, lb.Root).Set(commitLatency / 1e3)
```

Comment:
This (and below) is a milliseconds conversion and should remain `1e3`.

Reply:
Fixed. Replaced 1e3 with 1000 though.

Reply:
:+1: